modules/agentdestroy/agentdestroy.go

```diff
@@ -14,6 +14,7 @@ import (
 	"encoding/json"
 	"fmt"
 	"github.com/kardianos/osext"
+	"github.com/tudalex/seccomp-sandbox"
```

Comment:
The new repo is github.com/mozilla/mig-sandbox which should eventually contain sandbox code for other platforms as well (non-linux will just be empty for now)